addons/hr/static/tests/m2x_avatar_employee.test.js

```diff
@@ -117,17 +117,7 @@ test("many2one: click on an employee not associated with a user", async () => {
     const { env } = await makeMockServer();
     const employeeId = env["hr.employee.public"].create({ name: "Mario" });
     const avatarId = env["m2x.avatar.employee"].create({ employee_id: employeeId });
-    onRpc("web_read", (args) => {
-        expect.step(`web_read ${args.model} ${args.args[0]}`);
-        expect(args.kwargs.specification).toEqual({
-            employee_id: {
-                fields: {
-                    display_name: {},
-                },
-            },
-            display_name: {},
-        });
-    });
+    expect.step(`get_avatar_card_data hr.employee.public ${employeeId}`);
```

Comment:
Triggering a step unconditionally is not necessary. If we no longer need to assert the RPC is made, the corresponding `verifySteps` should be removed instead.